packages/react-composites/src/composites/CallComposite/adapter/CallingSoundSubscriber.test.ts

```diff
@@ -59,7 +59,7 @@ describe('Calling sound subscriber tests', () => {
     expect(audioMocks.Audio.play).toHaveBeenCalled();
   });
 
-  test('should not play sound when call is made to PSTN user', () => {
+  test('should play sound when call is made to PSTN user and stop when EarlyMedia', () => {
```

Comment:
Tests were updated to reflect the new changes. 
PSTN calls will now have ringing incorporated but will not have it incorporated in "EarlyMedia" state.
Contoso can easily override this behavior by providing no audio track for ringing.